Client.xcodeproj/project.pbxproj

```diff
@@ -5447,6 +5447,7 @@
 						CreatedOnToolsVersion = 6.1;
 						DevelopmentTeam = 2UYYSSHVUH;
 						LastSwiftMigration = 0830;
+						ProvisioningStyle = Automatic;
```

Comment:
We shouldn't use automatic Provisioning as in that case we won't be able to test push notifications on Dev builds